index.html

```diff
@@ -208,21 +208,28 @@ <h3>
           The {{ImageResource/label}} of an {{ImageResource}} is a [=string=]
           that provides an accessible name for the associated image.
         </p>
+        <p>
+          Implementors MUST [=calculate an accessible name=] for each
```

Comment:
Maybe "derive" or "compute", instead of "calculate"? "calculate" sounds very "math"